src/NetPanzer/Classes/AI/Astar.hpp

```diff
@@ -74,7 +74,6 @@ class AstarNode {
 enum { _path_merge_front, _path_merge_rear };
 
 class AstarNodePtrCompare
-    : public std::binary_function<const AstarNode *, const AstarNode *, bool> {
```

Comment:
This was gonna get fixed in https://github.com/netpanzer/netpanzer/pull/90 for the deprecation warning, but right now the build on macos arm64 is failing because of it.